packages/workflow/src/workflow.ts

```diff
@@ -840,18 +840,28 @@ function patchInternal(patchId: string, deprecated: boolean): boolean {
  *
  * @returns a boolean indicating whether the condition was true before the timeout expires
  */
-export function condition(timeout: number | string, fn: () => boolean): Promise<boolean>;
+export function condition(fn: () => boolean, timeout: number | string): Promise<boolean>;
```

Comment:
I know this came from another PR that's already merged and I know we're in alpha, but should be noted that this change is a silent runtime failure for a JS user. I know that this is a "TypeScript library" but most TypeScript libraries consider themselves JS libraries too. No worries of course in alpha.

Reply:
Next week it won't be okay to break the API like this anymore.
We're making the final changes now.